testing/mocker/create_common.py

```diff
@@ -0,0 +1,168 @@
+import pprint
```

Comment:
A general comment about the name of the file: `create_common.py` somehow doesn't feel apt as per the contents; perhaps `create_base.py`